sys-apps/onerng/onerng-3.6.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
```

Comment:
Doesn't it work with Python 3.6? If you haven't tested yet, then please do.

Reply:
It does, but the dependencies do not all support python3.6, like dev-python/python-gnupg.